scripts-available/CDB_Quota.sql

```diff
@@ -1,3 +1,22 @@
+CREATE OR REPLACE FUNCTION cartodb._CDB_total_relation_size(_schema_name TEXT, _table_name TEXT)
+RETURNS bigint AS
+$$
+BEGIN
+  IF EXISTS (
+      SELECT 1 FROM information_schema.tables
+      WHERE table_catalog = current_database()
+        AND table_schema = _schema_name
+        AND table_name = _table_name
+  )
+  THEN
+    RETURN pg_total_relation_size(format('"%s"."%s"', _schema_name, _table_name));
```

Comment:
If Javi is right, maybe doing like cartodbfication and [relying on exceptions](https://github.com/CartoDB/cartodb-postgresql/blob/master/scripts-available/CDB_CartodbfyTable.sql#L81) could be safer?

In any case this call should be really quick no?